flake.nix

```diff
@@ -14,6 +15,52 @@
       # Raw flake outputs (generally not system-dependent)
       flake = {
         devenvModules = import ./devenvModules;
+        nixosConfigurations.plutus-bench = nixpkgs.lib.nixosSystem {
+          system = "x86_64-linux";
+          specialArgs = { inherit inputs; };
+          modules = [
+            ./nixos/ax41-nvme.nix
+            {
+              system.stateVersion = "23.05";
+              hetzner-dedicated.nic = {
+                ipv4 = "65.108.236.230";
+                netmask4 = 26;
+                gateway4 = "65.108.236.193";
+                ipv6 = "2a01:4f9:1a:b0de::2";
+                netmask6 = 64;
+                mac = "a8:a1:59:a2:95:40";
+              };
+              users.users.root.openssh.authorizedKeys.keys = [ "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAID/fJqgjwPG7b5SRPtCovFmtjmAksUSNg3xHWyqBM4Cs shlevy@shlevy-laptop" ];
```

Comment:
Any need to add my machine's key here for access?

Reply:
Yes, let's add it for now (in the future would like to use a tf-generated key which is available in the shared state). Which key would you like to use?